hs_core/page_processors.py

```diff
@@ -275,6 +280,10 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
         g.is_user_member = user in g.gaccess.members
         if g.is_user_member:
             grps_member_of.append(g)
+    try:
```

Comment:
get_custom_citation?

Reply:
oh, you're getting the id and get_custom_citation only returns a string of the citation.  I assume you're using citation_id for deletion (haven't gotten through the whole review yet).  I'd prefer not passing around db ids for deletion... A resource can only have one citation, so the resource id should be enough for deleting the citation.